tools/depends/target/wayland-protocols/Makefile

```diff
@@ -0,0 +1,47 @@
+-include ../../Makefile.include
+DEPS=Makefile
+
+# lib name, version
+LIBNAME=wayland-protocols
+VERSION=1.7
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.xz
+
+ifeq ($(PLATFORM),)
+	# Building stand-alone
+	ROOT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+	PLATFORM = native
+	TARBALLS_LOCATION = $(ROOT_DIR)
+	BASE_URL := http://mirrors.kodi.tv/build-deps/sources
+	RETRIEVE_TOOL := curl -Ls --create-dirs -f -O
+	ARCHIVE_TOOL := tar --strip-components=1 -xf
+else
+	# Building as part of depends
+	DEPS += ../../Makefile.include
+endif
+
+# configuration settings
+CONFIGURE=./configure --prefix=$(PREFIX)
+
+CLEAN_FILES=$(ARCHIVE) $(PLATFORM)
+
+all: .installed-$(PLATFORM)
+
+$(TARBALLS_LOCATION)/$(ARCHIVE):
+	cd $(TARBALLS_LOCATION); $(RETRIEVE_TOOL) $(RETRIEVE_TOOL_FLAGS) $(BASE_URL)/$(ARCHIVE)
+
+$(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
```

Comment:
something like https://github.com/xbmc/xbmc/blob/94faa07/tools/depends/target/crossguid/Makefile#L48-L52 is missing